### DISCUSSION TITLE
How to target <u> with (custom?) prose selector?

### DISCUSSION THREAD

**Discussion by kobe-f8** [tailwindlabs/tailwindcss#11185] [Help]
Hello everyone, in one of my projects, I need to target the [<u\> element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/u).

My question is:
Is there a way to write custom tailwind prose classes that could select this tag, for example `prose-[u]:decoration-black`?

**Comment by wongjn** [comment]
You'd have to use a arbitrary variant (or as a custom plugin):

```
[&_:is(:where(u):not(:where([class~=not-prose]_*)))]:decoration-black
```

[Tailwind Play example](https://play.tailwindcss.com/31EKargPJr)
